src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10FeedFormatter.cs

```diff
@@ -133,15 +133,15 @@ internal static void ReadCategory(XmlReader reader, SyndicationCategory category
             {
                 while (reader.MoveToNextAttribute())
                 {
-                    if (reader.LocalName == Atom10Constants.TermTag && reader.NamespaceURI == string.Empty)
+                    if (reader.LocalName == Atom10Constants.TermTag && reader.NamespaceURI.Length == 0)
```

Comment:
For all of these NamespaceURI cases, how do we know they're not null?

Reply:
```suggestion
                    if (reader.LocalName == Atom10Constants.TermTag && reader.NamespaceURI == string.Empty)
```